### PR TITLE
Format and comment fw, mc, vtol, and ugv vehicle apps and default startup scripts

### PR DESCRIPTION
**Pull request by mcsauder**
Hi,

This PR consists of comment updates and whitespace changes, there is little to no substance in this PR,... It simply aims to improve readability and intent of the scripts.

Let me know if you have any questions or would prefer to dismiss this PR, which I am certainly ok with. :)

-Mark

### CODE REVIEW

File: ROMFS/px4fmu_common/init.d/rc.fw_defaults [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+# NOTE: Script variables are declared/initialized/unset in the rcS script.
+#
 
 set VEHICLE_TYPE fw
 
 if [ $AUTOCNF == yes ]
 then
 	#
-	# Default parameters for FW
+	# Default parameters for fixed wing uavs.
```
Comment (hamishwillee) [inline ROMFS/px4fmu_common/init.d/rc.fw_defaults]: UAV is an acronym

Comment (mcsauder) [reply ROMFS/px4fmu_common/init.d/rc.fw_defaults]: Haha!  Hamish! You get where I am coming from. :)  Let me know your preference!

**Comment by dakejahl** [comment]
I think he is suggesting you capitalize UAV ;)

File: ROMFS/px4fmu_common/init.d/rc.fw_defaults (hunk 1 above)
Comment (hamishwillee) [reply ROMFS/px4fmu_common/init.d/rc.fw_defaults]: I'm OK with this, but if you do any other tidy, suggest you do `# Default parameters for fixed wing UAVs.`

Comment (mcsauder) [reply ROMFS/px4fmu_common/init.d/rc.fw_defaults]: Excellent! I understand! Thank you!

**Pull request merged**